src/components/DownloadImageDialog/ApplicationInstructions.tsx

```diff
@@ -30,6 +30,9 @@ export const getUserOs = () => {
 	return 'Unknown';
 };
 
+const SECURE_BOOT_INSTRUCTION_START =
+	'Ensure there are no other USB keys are inserted. Power on the Generic x86_64 (GPT) with a keyboard connected.';
```

Comment:
NIT: I would prefer storing the start of the part that needs to be replaced eg `Press the F10 key while BIOS`.
So it will be visible in the code that we are replacing the BIOS related section with UEFI.